UKHomeOffice/cerberus-service
# COP-11456: Added test for Photograph addition (Issue target)

## Description
Added test for Photograph addition (Issue target)
## To Test
npm run cypress:test:local -- --spec cypress/integration/airpax/issue-aipax-tasks.spec.js

Should verify a photograph is added to TIS successfully
## Developer Checklist

\* Required
